src/components/Label/__mocks__/mockLabelExamples.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
```

Comment:
I think the vote was to start moving these examples into "Stories" instead of "__mocks__" - my apologies for the confusion on this.